### PR TITLE
Include Image component to form and Selector for States

### PR DESCRIPTION
**Pull request by emejia01**
This PR makes includes the add image component to the form page as well as changes the States entry field to a drop down selector.

### CODE REVIEW

File: vue-dash/src/components/sub-components/myListings-subpage/EditListing.vue [hunk 1]
```diff
@@ -19,14 +19,13 @@
             label="City"
             filled>
           </v-text-field>
-          <v-text-field class="listing-state-entry"
-            solo
-            label="State/Province"
-            filled>
-          </v-text-field>
+          <div class="divider"/>
```
Comment (NicolasCapparelli) [inline vue-dash/src/components/sub-components/myListings-subpage/EditListing.vue:22]: just FYI there's a Vuetify divider that you can use called v-divider if you didn't know

File: vue-dash/src/components/sub-components/myListings-subpage/EditListing.vue [hunk 2]
```diff
@@ -136,6 +141,22 @@ export default {
     items: ['1', '2', '3', '4', '5'],
     bedrooms: ['1', '2', '3', '4', '5'],
     baths: ['1', '2', '3'],
```
Comment (NicolasCapparelli) [inline vue-dash/src/components/sub-components/myListings-subpage/EditListing.vue:143]: Option for .5 bath?

File: vue-dash/src/components/sub-components/myListings-subpage/EditListing.vue [hunk 3]
```diff
@@ -169,4 +191,10 @@ export default {
   display: flex;
   justify-content: flex-end;
 }
+
+.divider{
+    width:20px;
```
Comment (NicolasCapparelli) [inline vue-dash/src/components/sub-components/myListings-subpage/EditListing.vue:196]: 4 spaces -> 2 spaces

File: vue-dash/src/components/sub-components/myListings-subpage/EditListing.vue (hunk 1 above)
Comment (emejia01) [reply vue-dash/src/components/sub-components/myListings-subpage/EditListing.vue:22]: Awesome! I'll definitely start using it! 👍

File: vue-dash/src/components/sub-components/myListings-subpage/EditListing.vue (hunk 3 above)
Comment (emejia01) [reply vue-dash/src/components/sub-components/myListings-subpage/EditListing.vue:196]: Will Change

**Pull request merged**